_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -3384,7 +3384,6 @@
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
 /obj/machinery/door/airlock{
 	name = "Auxiliary Storage Closet";
```

Comment:
Sorry, didn't realize I had to do that too. Was trying to minimize the stuff I delete in case it's something important